### ISSUE TITLE
Can't deploy without a GET / handler

### ISSUE THREAD

**Issue by grahamb** [architect/package#27]
**Describe the issue**
Possible duplicate of architect/deploy#22 

When a project has no get-index (`GET /`) handler, deploy fails.

**Steps to reproduce**
- `arc init` a new project
- remove the default get-index function and corresponding entry from `.arc`
- add another handler for a different path (GET /whatever-you-like)
- `arc deploy`

**Expected behavior**
It should deploy

** Actual behaviour**
Deploy fails:

```
$ arc deploy         
         App ⌁ course-outlines-lti-arc
      Region ⌁ ca-central-1
     Profile ⌁ sfu-its-lcs-canvas
     Version ⌁ Architect 6.0.14
         cwd ⌁ /Users/grahamb/code/sfu/course-outlines-lti-arc

⚬ Deploy Initializing deployment
  | Stack ... CourseOutlinesLtiArcStaging
  | Bucket .. my-deploy-bucket
⚬ Hydrate Hydrating dependencies in 1 path
✓ Hydrate Hydrated src/http/post-launch
  | npm ci: added 51 packages in 2.421s
⚬ Hydrate Hydrating app with shared files
✓ Hydrate Hydrated app with src/shared
✓ Success! Finished hydrating dependencies
Uploading to 9bd8d4d7420cff15b745a86c1ee8e5da  7167605 / 7167605.0  (100.00%)
Unable to upload artifact /Users/grahamb/code/sfu/course-outlines-lti-arc/node_modules/@architect/http-proxy/dist referenced by CodeUri parameter of GetIndex resource.
Parameter CodeUri of resource GetIndex refers to a file or folder that does not exist /Users/grahamb/code/sfu/course-outlines-lti-arc/node_modules/@architect/http-proxy/dist
deploy failed! 
```

** Additional information **

I don't have a `get-index` handler for this project because I'm using the `public` directory.

**Comment by ryanblock** [comment]
Released in Architect `6.0.16`!

**Comment by grahamb** [comment]
So far, so good here. Thanks, @ryanblock!

**Comment by grahamb** [comment]
… actually, spoke too soon. Still getting this error on 6.0.16.

```
         App ⌁ testapp
      Region ⌁ ca-central-1
     Profile ⌁ sfu-its-lcs-canvas
     Version ⌁ Architect 6.0.16
         cwd ⌁ /private/tmp/arc-public-directory-issue

⚬ Deploy Initializing deployment
  | Stack ... TestappStaging
  | Bucket .. ca-sfu-its-lcp-arc-test-deploy
✓ Hydrate Finished checks, nothing to hydrate
Uploading to 8135d47fd4bd610f65ef2119834ca84f  574 / 574.0  (100.00%)
Unable to upload artifact /private/tmp/arc-public-directory-issue/node_modules/@architect/http-proxy/dist referenced by CodeUri parameter of GetIndex resource.
Parameter CodeUri of resource GetIndex refers to a file or folder that does not exist /private/tmp/arc-public-directory-issue/node_modules/@architect/http-proxy/dist
```

Sample repo: https://github.com/grahamb/arc-public-directory-issue

**Comment by grahamb** [comment]
So, it works if `@architect/architect` is installed locally within the project. Is it required to be a local dependency?

**Comment by ryanblock** [comment]
> Is it required to be a local dependency?

It's definitely not a requirement! Thanks for reporting, that bug (diff from this bug, although the error is the same) is now fixed in `6.0.17`.
